src/libpgagroal/management.c

```diff
@@ -1610,6 +1613,27 @@ write_ssl(SSL* ssl, void* buf, size_t size)
    return 1;
 }
 
+static int
+write_header_info(char* header, bool start, char* command) {
+   char header_info[42] = " {sender: ";
+
+   strcat(header_info, command);
+   strcat(header_info, "version: ");
```

Comment:
I would remove the space after the colon, is it useful?

Reply:
Th spaces give a debug message like this one:
`2024-03-18 17:47:16 DEBUG management.c:1635 Header:  {sender: pgagroal version: 1.7.0}
`

Reply:
> Th spaces give a debug message like this one: `2024-03-18 17:47:16 DEBUG management.c:1635 Header: {sender: pgagroal version: 1.7.0} `

I see, but we should consider the header size as first, not the header human readability.
I'm not sure we want to send out an header longer than the payload, so I would rather add a single character for the indication of the fields, like `S:pgagroal;V:1.7.0` or even converting the version number into a digit (e.g., 1.7.0 could become 107000 and be sent as an int32).

Reply:
> I see, but we should consider the header size as first, not the header human readability.
As for now, the header has: 1 byte for `type`, 4 bytes for `slot` and 18-22 bytes in case the application name is `pgagroal` or `pgagroal-cli` respectively. Converting the version to int32 saves us 1 byte (if I'm not mistaken).

Reply:
> I see, but we should consider the header size as first, not the header human readability.

As for now, the header has: 1 byte for type, 4 bytes for slot and 18-22 bytes in case the application name is pgagroal or pgagroal-cli respectively. Converting the version to int32 saves us 1 byte (if I'm not mistaken).